pyaedt/edb_core/nets.py

```diff
@@ -246,7 +246,8 @@ def generate_extended_nets(
             Threshold of capacitor value. Search extended net across capacitors which has value higher than the
             threshold.
         exception_list : list, optional
-            List of components which bypass threshold check. The default is ``None``.
+            List of components which bypass threshold check. Components in the list are considerred as serial
+        components.The default is ``None``.
```

Comment:
```suggestion
            List of components to bypass when performing threshold checks. Components
            in the list are considered as serial components. The default is ``None``.
```
The four preceding descriptions should be changed to match the suggestions made earlier for these same parameters.